### PR TITLE
feed texts

### PR DESCRIPTION
**Pull request by kistitan**
New texts for activity feed.
Also please feel free to review any current English text

### CODE REVIEW

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_like_video">loved a video</string>
+    <string name="feed_title_like_videos">loved %1$d videos</string>
+    <string name="feed_title_like_status">loved a new status</string>
+    <string name="feed_title_like_statuses">loved %1$d status reports</string>
+    <string name="feed_title_like_picture">loved a picture</string>
+    <string name="feed_title_like_pictures">loved %1$d pictures</string>
+    <string name="feed_title_like_temp">%1$s loved a thingy</string>
+    <string name="feed_title_like_temps">%1$s loved %2$d thingies</string>
+    <string name="feed_title_like_unknown">Not yet supported Like Event</string>
+    <string name="feed_title_new_friend">became friend with a member</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values/strings.xml]: friends

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="settings_title_feed_auto_expand_like">New Loves</string>
+    <string name="settings_summary_feed_auto_expand_like">Auto expand feed stories about when a followed user loved something</string>
+    <string name="settings_title_feed_auto_expand_relation">New Relations</string>
+    <string name="settings_summary_feed_auto_expand_relation">Auto expand feed stories about when a followed user become friend with someone</string>
+    <string name="settings_title_feed_auto_expand_profile">Profile updates</string>
+    <string name="settings_summary_feed_auto_expand_profile">Auto expand feed stories about various profile related changes</string>
+    <string name="settings_title_feed_auto_expand_comment">New Comments</string>
+    <string name="settings_summary_feed_auto_expand_comment">Auto expand feed stories about user comments</string>
+    <string name="settings_title_feed_auto_expand_post">New Post</string>
+    <string name="settings_summary_feed_auto_expand_post">Auto expand feed stories about when a user post new a text</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values/strings.xml]: posted a new writing

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -183,4 +159,154 @@
...
     <string name="message_friend_decision_failed">FetLife App: Your decision about friendship could not be sent as you had ogged out before it could have been processed</string>
+
+    <!-- *** Feed texts *** -->
+
+    <string name="title_activity_feed">Feed</string>
+
+    <string name="feed_title_friends">%1$s became friend with %2$d users</string>
+    <string name="feed_title_new_post">uploaded a new writing</string>
+    <string name="feed_title_new_posts">posted %1$d new writings</string>
+    <string name="feed_title_new_grouppost">uploaded a new group writing</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: this is grou post, not group writing... same for one below this

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_new_groupposts">posted %1$d new group writings</string>
+    <string name="feed_title_new_wallpost">uploaded a new wall writing</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: this is wall post as well I believe

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_new_wallposts">posted %1$d new wall writings</string>
+    <string name="feed_title_group_join">joined to a group</string>
+    <string name="feed_title_group_joins">joined to %1$d groups</string>
+    <string name="feed_title_new_picture">uploaded a new picture</string>
+    <string name="feed_title_new_pictures">uploaded %1$d new pictures</string>
+    <string name="feed_title_like_writing">loved a writing</string>
+    <string name="feed_title_like_writings">loved %1$d writings</string>
+    <string name="feed_title_like_video">loved a video</string>
+    <string name="feed_title_like_videos">loved %1$d videos</string>
+    <string name="feed_title_like_status">loved a new status</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: "status update"

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_like_statuses">loved %1$d status reports</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: "status update"?

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_like_picture">loved a picture</string>
+    <string name="feed_title_like_pictures">loved %1$d pictures</string>
+    <string name="feed_title_like_temp">%1$s loved a thingy</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: lol

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_like_temps">%1$s loved %2$d thingies</string>
+    <string name="feed_title_like_unknown">Not yet supported Like Event</string>
+    <string name="feed_title_new_friend">became friend with a member</string>
+    <string name="feed_title_new_friends">became friend with %1$d members</string>
+    <string name="feed_title_new_follow">started to follow a member</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: `started to follow a member` should probably be `followed member`

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_new_follows">started to follow %1$d members</string>
+    <string name="feed_title_relation_unknown">Not yet supported relation event</string>
+
+    <string name="feed_title_rsvp_yes">is going to an event</string>
+    <string name="feed_title_rsvps_yes">is going to %1$d events</string>
+    <string name="feed_title_rsvp_maybe">is maybe going to an event</string>
+    <string name="feed_title_rsvps_maybe">is maybe going to %1$d events</string>
+    <string name="feed_title_new_comment">added a new comment</string>
+    <string name="feed_title_new_comments">added %1$d new comments</string>
+    <string name="feed_title_new_statuscomment">commented on a new status</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: status update

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_new_statuscomments">commented on %1$d status reports</string>
+    <string name="feed_title_new_videocomment">commented on a video</string>
+    <string name="feed_title_new_videocomments">commented on %1$d videos</string>
+    <string name="feed_title_new_groupcomment">commented on a group post</string>
+    <string name="feed_title_new_groupcomments">commented on %1$d group posts</string>
+    <string name="feed_title_new_status">added a new status</string>
+    <string name="feed_title_new_statuses">added %1$d status messages</string>
+    <string name="feed_title_new_postcomment">commented on a post</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: writing?

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_new_postcomments">commented on %1$d posts</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: writings?

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -183,4 +159,154 @@
...
+    <string name="feed_title_people_into_fetish">set a relation with fetish</string>
+    <string name="feed_title_people_into_fetishes">set relations with %1$d fetishes</string>
+    <string name="feed_caption_member_count" >%1$d members</string>
+
+    <string name="settings_title_feede">Feed Settings</string>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: `settings_title_feede`? what's feede? :D

**Comment by reneklacan** [comment]
@kistitan after you go through my and @rakvat's suggestions you should definitely ask @johnbaku for last review 👍

**Comment by Two-Finger** [comment]
Maybe align these strings with the website's?
I'm seeing some that definitely differ from what I would see if browsing FetLife.

**Pull request merged**